### PR TITLE
WWW-793: Closed captioning default display issues

### PR DESCRIPTION
**Pull request by colbytcook**
## Jira

https://pegadigitalit.atlassian.net/browse/WWW-793

## Summary

Updated the Brightcove video examples in Patternlab

## Details

* Added a video with localized translation to Patternlab to test closed captioning
* Updated the first play event to a more stable version (refer to https://docs.brightcove.com/brightcove-player/1.x/Player.html#firstplay-event)

## To Test

* Pull down the branch and start Bolt
* Open http://localhost:3000/pattern-lab/?p=components-accordion-content-variations
* Open the "Expand to see a video" tab and play the video, you should see captions present on the video
** (Optionally you can change the lang tag on the HTML to test the other languages, "it" for example should automatically play the Italian captions)

To test the ".one("play"..." adjustment repeat the process above in Safari and confirm you see the captions

### CODE REVIEW

File: packages/website-ui/styleguidekit-twig-default/views/partials/general-footer.twig [hunk 1]
```diff
@@ -105,12 +105,12 @@
         this.autoplayOnViewport();
       }
     })
-    thisPlayer.on('firstplay', function(player){
+    thisPlayer.one('play', function(player) {
       // NOTE: Safari does not immediately load the closed captioning menu on load, need to first wait until 'firstplay' event + 1/2 second wait time to avoid race conditions
```
Comment (remydenton) [inline packages/website-ui/styleguidekit-twig-default/views/partials/general-footer.twig:109]: Nitpick, the comment still mentions a 'firstplay' event.

**Review by remydenton — APPROVED** [review]
From the Brightcove docs 😂 :

> Not part of the HLS spec, and we're not sure if this is the best implementation yet, so use sparingly.

Good find!

**Review by cjwhitedev — CHANGES_REQUESTED** [review]
@colbytcook this wasn't working in safari for me. We should test in a few more environments and if it's only me I'll approve.

File: packages/website-ui/styleguidekit-twig-default/views/partials/general-footer.twig (hunk 1 above)
Comment (colbytcook) [reply packages/website-ui/styleguidekit-twig-default/views/partials/general-footer.twig:109]: Nice catch, I will update

Comment (colbytcook) [reply packages/website-ui/styleguidekit-twig-default/views/partials/general-footer.twig]: @remydenton this is updated

**Review by cjwhitedev — APPROVED** [review]

**Pull request merged**